backend/src/main/java/com/bakdata/conquery/resources/api/ConceptsProcessor.java

```diff
@@ -184,7 +184,7 @@ public ResolvedConceptsResult resolveFilterValues(AbstractSelectFilter<?> filter
 
 		QuickSearch<FilterSearchItem> search = filter.getSourceSearch();
 		if (search != null) {
-			result = createSourceSearchResult(filter.getSourceSearch(), Collections.singletonList(text), OptionalInt.of(50), FilterSearch.FilterSearchType.CONTAINS::score);
+			result = createSourceSearchResult(filter.getSourceSearch(), Collections.singletonList(text), OptionalInt.empty(), FilterSearch.FilterSearchType.CONTAINS::score);
```

Comment:
befüllt das den cache?

Reply:
Okay, ich habe irgendwie den cache nicht benutzt 🤔 hier muss ich nochmal ran

Reply:
@awildturtok ach nee ist alles okay, github hat mir igendwie nicht die ganze Datei durchsucht

Reply:
Die eigentliche Funktion/Lambda ist auch in der Cache definition